### ISSUE TITLE
JavadocTagContinuationIndentation: doesn't report violations for html elements

### ISSUE THREAD

**Issue by Gaurav-Punjabi** [checkstyle/checkstyle#8183]
https://checkstyle.org/config_javadoc.html#JavadocTagContinuationIndentation
1. Test.java:
```
gauravpunjabi@Gauravs-MacBook-Pro:~/Coding/Open Source/checkstyle$ javac Test.java 
gauravpunjabi@Gauravs-MacBook-Pro:~/Coding/Open Source/checkstyle$ cat -n Test.java 
     1  public class Test {
     2          /**
     3           * Some description here.
     4           *
     5           * @param a
     6           * Text description here. // violation
     7           * @param b
     8           * <b>Some description in bold</b> // OK
     9           */
    10          public void test(int a, int b) {}
    11  }
```
2. Config file:
```
gauravpunjabi@Gauravs-MacBook-Pro:~/Coding/Open Source/checkstyle$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <property name="charset" value="ASCII" />
    <property name="severity" value="error" />
    <property name="fileExtensions" value="java" />

    <module name="TreeWalker">
        <module name="JavadocTagContinuationIndentation" />
    </module>
</module>
```

3. Output:
```
gauravpunjabi@Gauravs-MacBook-Pro:~/Coding/Open Source/checkstyle$ java -jar target/checkstyle-8.32-SNAPSHOT-all.jar -c config.xml Test.java 
Starting audit...
[ERROR] /Users/gauravpunjabi/Coding/Open Source/checkstyle/Test.java:6: Line continuation have incorrect indentation level, expected level should be 4. [JavadocTagContinuationIndentation]
Audit done.
Checkstyle ends with 1 errors.
```

#### Expected
Line 8 should be reported as a violation, it is not correctly indented.

#### Explaination
JavadocTagContinutation does not report indentation violations for `HTML_ELEMENT`. As discussed in here https://github.com/checkstyle/checkstyle/pull/8002#discussion_r414565394 .
`HTML_ELEMENT` must also be properly intended to make the docs look good and easily readable.

**Comment by wltan** [comment]
I should be able to handle it under the scope of #8177, the root cause is similar to #5711.

```
D:\checkstyletest>type config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="severity" value="error"/>
    <module name="TreeWalker">
        <module name="JavadocTagContinuationIndentation">
          <property name="offset" value="4"/>
        </module>

    </module>
</module>

D:\checkstyletest>type test\Test.java
public class Test {
          /**
           * Some description here.
           *
           * @param a (text is needed here until #8002 is merged)
           * Text description here. // violation
           * @param b (text is needed here until #8002 is merged)
           * <b>Some description in bold</b> // violation
           */
          public void test(int a, int b) {}
}
D:\checkstyletest>java -jar -Duser.language=en -Duser.country=US checkstyle-8.32-SNAPSHOT-all.jar -c config.xml test\Test.java
Starting audit...
[ERROR] D:\checkstyletest\test\Test.java:6: Line continuation have incorrect indentation level, expected level should be 4. [JavadocTagContinuationIndentation]
[ERROR] D:\checkstyletest\test\Test.java:8: Line continuation have incorrect indentation level, expected level should be 4. [JavadocTagContinuationIndentation]
Audit done.
```

**Comment by Gaurav-Punjabi** [comment]
> I should be able to handle it under the scope of #8177, the root cause is similar to #5711.

@wltan That would be great 👍

**Comment by romani** [comment]
possible fix is merged in PR for #5711
